contracts/cw-ics721-bridge/Cargo.toml

```diff
@@ -13,20 +13,21 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cosmwasm-std = { version = "1.1", features = ["stargate"] }
+cosmwasm-std = { version = "1.1", features = ["ibc3"] }
 cosmwasm-schema = "1.1"
-cw-storage-plus = "0.15"
-cw-utils = "0.15"
-cw2 = "0.15"
-cw721 = "0.15"
-cw721-base = { version = "0.15", features = ["library"] }
+cw-storage-plus = "1.0.1"
+cw-utils = "1.0.1"
```

Comment:
This version conflicts with launchpad (main repo) on Stargaze so might want to do updated version here: 
From launchpad: 
cw-utils = "0.16.0"

The same should apply for cw2. Would check out launchpad repo for reference.

Reply:
does launchpad depend on this contract? these are the latest versions from cw-plus, so if this doesn't immediately break any dependencies i figure we may as well update